README.md

```diff
@@ -1,100 +1,112 @@
-* `--local-host` proxy to a hostname other than localhost
+- `--subdomain` request a named subdomain on the localtunnel server (default is random characters)
+- `--local-host` proxy to a hostname other than localhost
 
-You may also specify arguments via env variables.  E.x.
+You may also specify arguments via env variables. E.x.
 
 ```
 PORT=3000 lt
 ```
 
-## API ##
+## API
 
 The localtunnel client is also usable through an API (for test integration, automation, etc)
 
-### localtunnel(port [,opts], fn)
+### localtunnel(port [,options][,callback])
 
-Creates a new localtunnel to the specified local `port`. `fn` will be called once you have been assigned a public localtunnel url. `opts` can be used to request a specific `subdomain`.
+Creates a new localtunnel to the specified local `port`. Will return a Promise that resolves once you have been assigned a public localtunnel url. `options` can be used to request a specific `subdomain`. A `callback` function can be passed, in which case it won't return a Promise. This exists for backwards compatibility with the old Node-style callback API. You may also pass a single options object with `port` as a property.
 
-```javascript
-var localtunnel = require('localtunnel');
+```js
+const localtunnel = require('@chromaui/localtunnel');
 
-var tunnel = localtunnel(port, function(err, tunnel) {
-    if (err) ...
+(async () => {
+  const tunnel = await localtunnel({ port: 3000 });
 
-    // the assigned public url for your tunnel
-    // i.e. https://abcdefgjhij.localtunnel.me
-    tunnel.url;
-});
+  // the assigned public url for your tunnel
+  // i.e. https://abcdefgjhij.localtunnel.me
+  tunnel.url;
 
-tunnel.on('close', function() {
+  tunnel.on('close', () => {
     // tunnels are closed
-});
+  });
+})();
 ```
 
-### opts
+#### options
 
-* `subdomain` A *string* value requesting a specific subdomain on the proxy server. **Note** You may not actually receive this name depending on availability.
-* `local_host` Proxy to this hostname instead of `localhost`. This will also cause the `Host` header to be re-written to this value in proxied requests.
+- `port` [required] The local port number to expose through localtunnel.
+- `subdomain` A _string_ value requesting a specific subdomain on the proxy server. **Note** You may not actually receive this name depending on availability.
+- `local_host` Proxy to this hostname instead of `localhost`. This will also cause the `Host` header to be re-written to this value in proxied requests.
 
 ### Tunnel
 
 The `tunnel` instance returned to your callback emits the following events
 
-|event|args|description|
-|----|----|----|
-|request|info|fires when a request is processed by the tunnel, contains _method_ and _path_ fields|
-|error|err|fires when an error happens on the tunnel|
-|close||fires when the tunnel has closed|
+| event   | args | description                                                                          |
```

Comment:
Is this just for README formatting or does it have an effect on the generated markdown?

Reply:
I use Prettier, it automatically formats markdown like that. Doesn't have any further effect besides readability.